ui/src/main/resources/FileManager/FileManagerTranslations.pt_BR.xml

```diff
@@ -0,0 +1,90 @@
+<xwikidoc>
+  <web>FileManager</web>
+  <name>FileManagerTranslations</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <parent>FileManagerClass</parent>
+  <creator>xwiki:XWiki.Admin</creator>
+  <author>xwiki:XWiki.Admin</author>
+  <customClass/>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <creationDate>1355094000000</creationDate>
+  <date>1355094000000</date>
+  <contentUpdateDate>1355094000000</contentUpdateDate>
+  <version>1.1</version>
+  <title/>
+  <template/>
+  <defaultTemplate/>
+  <validationScript/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <object>
+    <class>
+      <name>XWiki.TranslationDocumentClass</name>
```

Comment:
This object doesn't belong here. It should be already present on the default translation.